src_firmware/src/driver.h

```diff
@@ -5,34 +5,25 @@
 
 #include <zephyr/kernel.h>
 
-#define MIN_TO_MS 60000
-#define RPM_TO_MRPM 1000
+
 
 /// @brief Error code definitions for motor driver
 enum error_codes {
 	SUCCESS = 0,
 	NOT_INITIALISED = 1,
```

Comment:
```suggestion
	ERR_NOT_INITIALISED = 1,
```
please add `ERR_` prefix to all error codes to make them more readable

Reply:
PR for error code refactor is already underway. This will be a part of it